### PR TITLE
Change of bootstrap switch

### PR DESCRIPTION
**Pull request by Allexander203**
Changed the bootstrap swich : added data-on-text and data-off-text to the swiches.

### CODE REVIEW

**Review by RMadjev — CHANGES_REQUESTED** [review]
First of all use English only. Also do these changes in single commit and describe better what you did in the commit. I good practice for the commit message is to use this: "If applied, this commit will _your subject line here_". ref. to this article https://chris.beams.io/posts/git-commit/

**Review by atodorov — CHANGES_REQUESTED** [review]
Change Title & description to English!

**Review by RMadjev — CHANGES_REQUESTED** [review]
Again you used 4 commits. We want just one with informative descriptive message.

**Review by atodorov — CHANGES_REQUESTED** [review]
HTML changes look good but you have failing tests which need to be fixed:
https://github.com/kiwitcms/Kiwi/pull/2028/checks?check_run_id=1297210061

File: tcms/testplans/tests/test_new_plan.py [hunk 1]
```diff
@@ -43,31 +43,31 @@ def setUp(self):
 
     def test_plan_new_get(self):
         response = self.client.get(self.location)
-
+
```
Comment (atodorov) [inline tcms/testplans/tests/test_new_plan.py:46]: flake8 complains about this: `W293 blank line contains whitespace`

**Review by atodorov — CHANGES_REQUESTED** [review]

**Review by atodorov — DISMISSED** [review]

**Pull request merged**